libs/pika/type_support/include/pika/type_support/equality.hpp

```diff
@@ -84,4 +84,4 @@ namespace pika { namespace traits {
             typename std::decay<T>::type>
```

Comment:
All of this can also go into `detail`. In cases where there's a name class between the "implementation" base class in `detail` and the main template, I've been renaming the base classes to have an `_impl` suffix. I think that would work here as well.

Reply:
Yes I did it in one case but I was not sure about it. Now it's clear thanks!